src/zoning-district/zoning-district.repository.ts

```diff
@@ -39,18 +45,20 @@ export class ZoningDistrictRepository {
     }
   }
 
-  async findByUuid(uuid: string): Promise<FindByUuidRepo | undefined> {
+  async findByUuid(id: string): Promise<FindByUuidRepo | undefined> {
     try {
       return await this.db.query.zoningDistrict.findFirst({
         columns: { wgs84: false, liFt: false },
-        where: eq(zoningDistrict.id, uuid),
+        where: eq(zoningDistrict.id, id),
       });
     } catch {
       throw new DataRetrievalException();
     }
   }
 
-  async findClassesByUuid(id: string) {
+  async findClassesByUuid(
+    id: string,
+  ): Promise<FindClassesByIdRepo | undefined> {
```

Comment:
We can remove `undefined` since we'll be returning a list (and if we don't find any we'll return an empty list instead of `undefined`)